array-manipulation-1/src/compact.js

```diff
@@ -0,0 +1,13 @@
+/* exported compact */
+function compact(array) {
+  const newArr = [];
+  array.map((arr) => {
+    if (arr) {
+      newArr.push(arr);
+    }
+    return newArr;
+  });
+  return newArr;
+}
```

Comment:
I would not use `map` here & instead would use a normal `for` loop. `map` generates a new array, but since you created `newArr` already and you're just pushing into it, there's no need to use `map`, because you aren't actually even doing anything the the array that gets `return`ed from `map`